client/src/hooks/queries/useGetAmbiguousResults.ts

```diff
@@ -112,9 +112,13 @@ export function useGetMatchedResults(names: string[], type: ResultTypes) {
   return useQuery(
     key,
     async () => {
-      const res = await graphQLClient.request(requestQuery, {
-        searchTerms: names,
-      });
+      const res = await graphQLClient.request(
+        requestQuery,
+        {
+          searchTerms: names,
+        },
+        { 'dgidb-client-name': 'dgidb-frontend' }
```

Comment:
You might want to double check that it works as expected, but it looks like (based on this example [here](https://github.com/graffle-js/graffle/blob/graphql-request/examples/legacy/request-authentication-via-http-header.ts#L5)) you can set an HTTP Header when your initialize the graphql client.

That might be better than stapling it into each query individually.

Reply:
@acoffman yeah, I was a little unsure of whether we wanted to set different headers for different queries and to filter out noisier or less informative ones. We should probably set a base one at client initialization and then maybe use an additional one for chosen queries if we go that route.